README.md

```diff
@@ -1,41 +1,67 @@
-############################################
 # NF1 Inactivation Classifier for Glioblastoma
-#
-## (C) 2016 Gregory Way, Robert Allaway, Rebecca Barnard, Stephanie Bouley, Brian O'Roak, Camilo Fadul, Yolanda Sanchez, and Casey Greene
-############################################
+
+**(C) 2016 Gregory Way, Robert Allaway, Rebecca Barnard,
```

Comment:
Could make code climate happy with a header here. Looks like you had a header before, so this might be the way to go.